File: serenity-app/src/main/java/us/nineworlds/serenity/injection/modules/LoginModule.java
package us.nineworlds.serenity.injection.modules;

import toothpick.config.Module;
import us.nineworlds.serenity.ui.activity.login.LoginUserPresenter;

public class LoginModule extends Module {

  public LoginModule() {
    bind(LoginUserPresenter.class).to(LoginUserPresenter.class);
  }

}

File: serenity-app/src/main/java/us/nineworlds/serenity/injection/modules/LoginModule.kt
package us.nineworlds.serenity.injection.modules

import toothpick.config.Module
import us.nineworlds.serenity.ui.activity.login.LoginUserPresenter

class LoginModule : Module() {
    init {
        bind(LoginUserPresenter::class.java).to(LoginUserPresenter::class.java)
    }
}
